konspekt_pracy_magisterskiej.md

```diff
@@ -0,0 +1,59 @@
+﻿<![endif]-->
+
+**"Perception of aspiration in English"**
+
+**Abstract**
+
+**Introduction**
+
+Aim of the paper, thesis, introduction to subject, summary of chapters and subchapters.
+
+**Chapter 1 Theory**
+
+1. **Perceptual vs phonetic level**
+In this subchapter I would like to discuss the notion of both phonetic and perceptual level due to the fact that they differ largely i.e. the perception of heard speech sounds may vary as far as listeners are concerned, therefore, I would like to detailedly describe each of these levels. 
+
+2. **Aspiration**
+In this subchapter I would like to discuss the notion of phonological aspiration i.e. explain what it is, give an insight into environment for aspiration and describe the process of it. 
+2.1. **Manner of articulation - stop consonants**
+In this subsubchapter I would like to concentrate on manner of articulation i.e. on the configuration and interaction of the articulators (speech organs such as tongue, lips etc) when making a speech sound. Particularly, I will focus on English voiceless plosives and the concept of their manner.
+2.2. **Place of articulation - stop consonants**
+In this subsubchapter I would like to discuss the notion of the place of articulation i.e. the point of contact where an obstruction occurs in the vocal tract between an articulatory gesture, an active articulator (typically some part of the tongue), and a passive location (typically some part of the roof of the mouth. Considering the place of articulation, I will concentrate only on stop consonants.
+2.3. **Allophonic rules**
+In this subsubchapter I would like to extensively introduce the concept of allophpnic rules in which aspiration occurs. 
+
+3. **Speech perception**
+In this subchapter I would like to focus on the term of speech perception (how the sounds are heard, interpreted and understood) as well as the process of it. What is more, I would like to also concentrate on such field as cognitive psychology and perception in psychology which are closely related to phonology and phonetics.
+3.1. **VOT (Voiced Onset Time)**
+In this subsubchapter I would like to discuss the notion of voice onset time which refers to the time interval between the release of a stop and the beginning of vocal fold vibration. Furthermore, I would like to discuss types of VOT and give an insight into what positive, negative and zero VOT mean. 
+
+4.**Interpreting spectograms**
```

Comment:
Space is missing after the period. This is why the text is not rendering correctly. Look at https://github.com/nlp-ug-2018-19/MA-Thesis-Overview/blob/NikolaGrubecka-patch-2/konspekt_pracy_magisterskiej.md 
..and a typo specto > spectro